app/src/main/java/cm/aptoide/pt/analytics/FirstLaunchAnalytics.java

```diff
@@ -74,6 +87,88 @@ public void sendFirstLaunchEvent(String utmSource, String utmMedium, String utmC
         FIRST_LAUNCH, AnalyticsManager.Action.OPEN, "");
   }
 
+  private void sendPlayProtectEvent() {
+    SafetyNet.getClient(context)
+        .listHarmfulApps()
+        .addOnCompleteListener(task -> {
+          String isActive = "false";
```

Comment:
To make this cleaner i would still use boolean objects and then i would convert to true or false where i am putting the value.
Example: isActive ? "true" : "false"